Add vega magnitudes (and fluxes) to simulations

karllark opened the pull request:
This PR adds vega magnitudes and physical fluxes to the simulation output.  Docs on simulations updated.
Closes #594.

Partially addresses #593 in that the completeness is now done as the max across all filters for each model.

In addition, there are fixes for the tests so that they run.  Moved the cut of LogL < -9.99 in the isochrones to be compatible with how the beast is run *and* the tested.

Finally, re-enabled running some beast scripts via `beast scriptname`.  This was accidentally removed when the update to APE17 was merged (#582).

TODO: fix the rest of the regression tests.  @lea-hagen : need to regenerate those that you generated the files for (test_remove_filters, test_star_type_probability_all_params, test_compare_spec_type_inFOV).   Also test_ast_pick_models (who did this test file? @cmurray-astro ?)  There are a couple more tests that I wrote - going to see if I can change them so that they are not specific to the exact numbers (too much manual work to update them when we change things upstream from them).

cmurray-astro commented:
I am pretty sure that it was not me on test_ast_pick_models, but happy to help with this if needed

karllark commented:
@lea-hagen : If you could have a look at this PR in the next day or so, that would be great.  I'll work on fixing some of the other test failures in a separate PR.

karllark commented:
Failing regression tests will be addressed in separate PRs.  See #598.

lea-hagen commented on docs/simulations.rst:
```diff
@@ -25,21 +26,22 @@ the script.  The output filename is also required.  Note that the extension
-`--compl_filter` parameter.  By default, the SEDs are randomly chosen, and
-weighted by their grid+prior weights; this can be changed with the
-`--weight_to_use` parameter.
+The SEDs picked weighted by the product of the grid+prior weights
+and the completeness from the noisemodel.  The grid+prior weights can be replaced
+with either grid or prior weights by explicitly setting the `--weight_to_use`
+parameter.
```
I think this needs another word - "The SEDs **are** picked weighted by..."

lea-hagen commented on docs/simulations.rst:
```diff
@@ -25,21 +26,22 @@ the script.  The output filename is also required.  Note that the extension
-`band_rate` and the units are normalized Vega fluxes (to match how
-the observed data are given).
+from the physicsgrid file.  The simulated observations in each band are given
+in as `band_flux` in physical units (ergs cm^-2 s^-1 A^-1),
```
small typo - remove "in"

karllark commented:
I addressed the two docs comments.  If you are happy with my fixes, please approve this PR (trying to be more formal in PR reviews as this is a good practice).

lea-hagen commented:
Oh weird, it wasn't until after I approved it that it showed the merge conflict.  I'm happy to look at it again once that's resolved, if you'd like.

karllark commented:
The merge conflict was minor.  The merge conflict happened because I merged your PR in.  I just rebased and fixed the conflict.  Minor update.   Have a look, but not required.

lea-hagen commented:
That's what I figured.  Looks fine!
